nexus/tests/integration_tests/endpoints.rs

```diff
@@ -872,6 +932,14 @@ lazy_static! {
             ],
         },
 
+        VerifyEndpoint {
+            url: "/by-id/images/{id}",
+            visibility: Visibility::Protected,
+            allowed_methods: vec![
+                AllowedMethod::GetUnimplemented,
```

Comment:
This lookup isn't implemented due to the underlying create/fetch not being implemented. Project images are still stubs at this point.